<launch>  
    <node pkg="bitten" type="can_driver_node" name="can_driver_node"/>
    
    <node pkg="bitten" type="manual_node" name="manual_node"/>

    <node pkg="bitten" type="commander_node" name="commander_node"/>

    <node pkg="bitten" type="movement_node" name="movement_node"/>

    <arg name="robot_ip" default="10.82.18.254"/>
    <param name="robot_ip_address" type="str" value="$(arg robot_ip)" />

    <include file="$(find staubli_val3_driver)/launch/robot_state.launch">
        <arg name="robot_ip" value="$(arg robot_ip)" />
    </include>

    <include file="$(find staubli_val3_driver)/launch/motion_streaming_interface.launch">
        <arg name="robot_ip" value="$(arg robot_ip)" />
    </include>
</launch>
